lib/urlapi-int.h

```diff
@@ -30,4 +30,9 @@ bool Curl_is_absolute_url(const char *url, char *scheme, size_t buflen);
 char *Curl_concat_url(const char *base, const char *relurl);
 size_t Curl_strlen_url(const char *url, bool relative);
 void Curl_strcpy_url(char *output, const char *url, bool relative);
+
+#ifdef DEBUGBUILD
+CURLUcode parse_port(struct Curl_URL *u, char *hostname);
```

Comment:
Maybe (even if only for debug builds) still prefix this with `Curl_` to make it less likely for name collisions with something else?

Reply:
Thats probably a good idea, can't hurt to rename it `Curl_parse_port()` to have matching suspenders with the belt.